src/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SessionEndReasons.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Win32
+{
+    /// <devdoc>
```

Comment:
Do we care about cleaning up all these <devdoc> and <para> things to just have <summary>?

Reply:
This is used across the repo in ported code, so I didn't make the change.  I think if we want to get rid of them we should have a tool to do so and run it over the entire repo.